.eslintrc.js

```diff
@@ -142,6 +142,7 @@ module.exports = {
       'error',
       { argsIgnorePattern: '^_', ignoreRestSiblings: true },
     ],
+    '@typescript-eslint/no-unused-expressions': 'error',
```

Comment:
I would still like to have `allowShortCircuit: false` since it will forbid code style in favour of `if` statements, but if you dislike just click on the `Commit suggestion` 😜